python/pyhie/allspark/model/qwen_v30.py

```
@@ -2,6 +2,7 @@
  Copyright (c) Alibaba, Inc. and its affiliates.
  @file    qwen_v30.py
 '''
+import struct
```

Flake8 (7.3.0)

[error] 'struct' imported but unused

(F401)